### PR TITLE
profiles/base/package.use.stable.mask: mask USE flags for testing (was: media-gfx/alembic: fix boost not found error)

### PR DESCRIPTION
**Pull request by waebbl**
Fixes an error with >=cmake-3.11* where boost is not found due to changes in cmake syntax.

Closes: https://bugs.gentoo.org/667728

The `package.use.mask` change masks two USE flags for testing. I can't test them myself, because I don't have access to the needed software. But to let people know about the plugins, I decided to p.u.mask them, rather than removing them.

### CODE REVIEW

**Comment by mgorny** [comment]
@SoapGentoo, you're the boost expert here, could you take a look?

File: profiles/base/package.use.mask [hunk 1]
```diff
@@ -7,6 +7,10 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Bernd Waibel <waebbl@gmail.com> (11 Nov 2018)
+# masked for testing
```
Comment (a17r) [inline profiles/base/package.use.mask]: why do you mask existing flags for everyone?

File: media-gfx/alembic/alembic-1.7.9.ebuild [hunk 2]
```diff
@@ -14,10 +14,7 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="BSD"
 
 SLOT="0"
-# TODO: ~x86 currently depends on new =dev-python/pyilmbase-2.3.0 which has
-# ~x86 keyword. As soon as it's updated in the tree, the keyword can be
-# added here.
-KEYWORDS="~amd64"
+KEYWORDS="~amd64 ~x86"
```
Comment (a17r) [inline media-gfx/alembic/alembic-1.7.9.ebuild]: keywording should be left to x86 arch team, which at least is going to make an appropriate separate commit.

Comment (waebbl) [reply media-gfx/alembic/alembic-1.7.9.ebuild]: I tested it in a 32-bit chroot, because I don't have a real ~x86 machine available.

File: profiles/base/package.use.mask (hunk 1 above)
Comment (a17r) [reply profiles/base/package.use.mask]: so, the other part of this PR was merged now. I still find it odd to mask these flags after they have been available to the odd user in ~arch for a long time already. stable-mask them in case you planned to stabilise alembic, otherwise leave it alone?

Comment (waebbl) [reply profiles/base/package.use.mask]: I hope I can stabilize this package at some time, so I moved the masking to package.use.stable.mask, hoping this being the correct location. If it's not, feel free to ping me again.

File: profiles/base/package.use.stable.mask [hunk 3]
```diff
@@ -4,6 +4,10 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Bernd Waibel <waebbl@gmail.com> (13 May 2019)
+# needs build and runtime testing
```
Comment (a17r) [inline profiles/base/package.use.stable.mask]: That's not a particularly good mask message even if you'll find it used in many places (but it is more or less implied anyway by being in this file; or `~arch`, for that matter). I couldn't help but notice the ebuild telling that also `prman` is still experimental. Should it also be stable-masked then? The message should then contain the actual reason for the masking.

File: profiles/base/package.use.stable.mask [hunk 4]
```diff
@@ -4,6 +4,10 @@
...
+>=media-gfx/alembic-1.7.9 arnold maya
```
Comment (a17r) [inline profiles/base/package.use.stable.mask]: no need to constrain it to this min version. and no versions are currently stable anyway.

File: profiles/base/package.use.stable.mask (hunk 3 above)
Comment (waebbl) [reply profiles/base/package.use.stable.mask]: Thanks for the hint. I know it's not a good message, but I just looked at the common way in the file. Not a particularly good habit on my side either... I changed the message to hopefully be more meaningful.
The `prman` USE flag is told experimental in the ebuild, but it could theoretically be removed, and I'm probably doing this at the next bump. I have a renderman ebuild in my overlay and can test the addon with this. It has a basic working at least, but I don't had the time to thoroughly test it, so the info on being experimental is still there.

**Comment by waebbl** [comment]
It's probably a good time to get myself into how to stabilize a package. It has been around for quite some time, without many bugreports, so yes, I'm planning to stabilize this soon. AFAIR the usual time-span is about 4-6 weeks without bugs.

**Comment by a17r** [comment]
30 days in `~arch`, no open bugs.

**Comment by waebbl** [comment]
So just no open bugs? I thought no bugs at all during the 30 days period.
